sdk/eventhub/Azure.Messaging.EventHubs/src/Primitives/PartitionReceiver.cs

```diff
@@ -251,7 +257,9 @@ public class PartitionReceiver : IAsyncDisposable
             InitialPosition = eventPosition;
             DefaultMaximumWaitTime = options.DefaultMaximumReceiveWaitTime;
             RetryPolicy = options.RetryOptions.ToRetryPolicy();
+#pragma warning disable CA2214 // Do not call overridable methods in constructors. This internal method is virtual for testing purposes.
```

Comment:
nit:  Do you mind adding a leading space to this, just to give some visual separation before the pragma block?